src/RegExLib.Core/Entities/Author.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using RegExLib.Core.Events;
+using RegExLib.Core.Interfaces;
+using RegExLib.SharedKernel;
+
+namespace RegExLib.Core.Entities
+{
+    public class Author : BaseEntity
+    {
+        public string FullName { get; set; }
+        public string Username { get; set; }
+        public string UserId { get; set; }
+        public List<Expression> Expressions { get; private set; } = new List<Expression>();
```

Comment:
This could be better encapsulated by following the [pattern here](https://github.com/ardalis/ddd-guestbook/blob/c52c353fcef932707873841262563128adce9077/src/CleanArchitecture.Core/Entities/Guestbook.cs#L10-L15). This prevents collaborators from adding directly to the Expressions list, which could circumvent validation, events, or other logic related to adding Expressions.

Reply:
this already changed to 

```csharp
        public string FullName { get; }
        public string Username { get; }
        public string UserId { get; }
        public List<Expression> Expressions { get; private set; } = new List<Expression>();
```

Reply:
Done 
```csharp
        private readonly List<Expression> _expressions = new List<Expression>();
        public IEnumerable<Expression> Expressions => new ReadOnlyCollection<Expression>(_expressions);
```

Reply:
Yup. Like this: https://ardalis.com/encapsulated-collections-in-entity-framework-core/. Looks good.